src/Internal/Payment/Change.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Class Change
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal\Payment;
+
+/**
+ * A change object used for logging.
+ * Key, former and current values are set as string.
+ */
+class Change {
+
+	/**
+	 * Identifier or key for the changed value.
+	 *
+	 * @var string
+	 */
+	private $key;
+
+	/**
+	 * Former value for the key.
+	 *
+	 * @var mixed
+	 */
+	private $old_value;
+
+	/**
+	 * Current value, updated aganist the key.
```

Comment:
There are also 2-3 other similar mistypos in this file.

```suggestion
	 * Current value, updated against the key.
```

Reply:
Done.